src/torchcodec/_core/_metadata.py

```diff
@@ -88,14 +88,22 @@ def duration_seconds(self) -> Optional[float]:
         fall back to ``duration_seconds_from_header``.
```

Comment:
Similarly to what was done for the `num_frames` properties, we can slightly update the docstring above to describe the new fall-back